frontend/src/pages/account/Create.vue

```diff
@@ -91,8 +102,12 @@ export default {
             return (this.input.email && !this.errors.email) &&
                    (this.input.username && !this.errors.username) &&
                    this.input.password.length >= 8 &&
+                   (this.input.join_reason) &&
```

Comment:
This line breaks sign-up for instances without posthog, as `join_reason` is never set to anything so the sign-up button is never enabled.

Reply:
In fact, it is broken for all instances. Even with posthog enabled, the button never becomes enabled.